### PR TITLE
gallery-dl: init at 1.8.2

### PR DESCRIPTION
**Pull request by dawidsowa**
###### Motivation for this change

Add  https://github.com/mikf/gallery-dl

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/gallery-dl/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    sha256 = "6c5995b7d24dfaae7bdf71b9261e5044b01adbd5d5302aaff9ac4a30bbceedb6";
+  };
+
+  doCheck = false;
+  propagatedBuildInputs = with python3Packages; [ requests ];
+
+    meta = {
+    description = "Command-line program to download image-galleries and -collections from several image hosting sites";
+    homepage = https://github.com/mikf/gallery-dl;
+    license = lib.licenses.gpl3;
```
Comment (aanderse) [inline pkgs/applications/misc/gallery-dl/default.nix]: The code indicated GPLv2.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -1133,6 +1133,11 @@
     email = "davor@davor.se";
     github = "davorb";
     name = "Davor Babic";
+  };
+   dawidsowa = {
```
Comment (aanderse) [inline maintainers/maintainer-list.nix]: Please include this in a separate commit

File: pkgs/applications/misc/gallery-dl/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "6c5995b7d24dfaae7bdf71b9261e5044b01adbd5d5302aaff9ac4a30bbceedb6";
+  };
+
+  doCheck = false;
+  propagatedBuildInputs = with python3Packages; [ requests ];
+
+    meta = {
```
Comment (aanderse) [inline pkgs/applications/misc/gallery-dl/default.nix]: Please align this back 2 spaces

**Review by aanderse — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (Lassulus) [inline maintainers/maintainer-list.nix]: so close! but please remove 1 space before your name

File: pkgs/applications/misc/gallery-dl/default.nix [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    inherit pname version;
+    sha256 = "6c5995b7d24dfaae7bdf71b9261e5044b01adbd5d5302aaff9ac4a30bbceedb6";
+  };
+
+  doCheck = false;
+  propagatedBuildInputs = with python3Packages; [ requests ];
+
+  meta = {
+    description = "Command-line program to download image-galleries and -collections from several image hosting sites";
+    homepage = https://github.com/mikf/gallery-dl;
```
Comment (Lassulus) [inline pkgs/applications/misc/gallery-dl/default.nix]: We probably will remove the unquoted url syntax in the near future: https://github.com/NixOS/rfcs/pull/45
And sorry for the delay, I didn't get a notification when you force pushed. Can you ping me with @lassulus after the next push and I will merge this :)

Comment (Lassulus) [reply pkgs/applications/misc/gallery-dl/default.nix]: Ah well, I guess it's not finalized yet and we can change it later. gonna merge it now

**Pull request merged**